src/cpu/wino_reorder.hpp

```diff
@@ -187,10 +187,10 @@ struct wino_reorder_t : public primitive_impl_t {
             out_data_t *__restrict _out
                     = tmp_wei + (iic * nb_oc_ + ob) * oc_block_;
 
-            parallel_nd(size_wspace_, [&](int i) { wspace[i] = 0.f; });
```

Comment:
I believe the claim in the commit message, but how does this affect other cases? It also would be interesting to know how it affects different compilers.

Reply:
We don't have performance data for other workloads, but it spawns an excessive amount of very small tasks, which have a too high overhead. We tested this with ICC, but I don't expect significantly different results with other compilers. With parallel_nd we have an initialization time of more than 800 ms for a U-Net-like architecture, which is too much. A 4x speedup by avoiding parallel execution indicates that there's barely any parallelism available in these loops, and the overhead of parallel_nd is huge.